Hazel/FewerThreads/ThreadLimitedUdpConnectionListener.cs

```diff
@@ -334,13 +340,14 @@ protected virtual void ReadCallback(MessageReader message, IPEndPoint remoteEndP
             connection.HandleReceive(message, bytesReceived);
         }
 
-        internal void SendDataRaw(byte[] response, IPEndPoint remoteEndPoint)
+        internal void SendDataRaw(SmartBuffer response, IPEndPoint remoteEndPoint)
         {
             QueueRawData(response, remoteEndPoint);
         }
 
-        protected virtual void QueueRawData(ByteSpan span, IPEndPoint remoteEndPoint)
+        protected virtual void QueueRawData(SmartBuffer span, IPEndPoint remoteEndPoint)
         {
+            span.AddUsage();
             this.sendQueue.TryAdd(new SendMessageInfo() { Span = span, Recipient = remoteEndPoint });
```

Comment:
Naw, send queue joins, so the sender thread should best effort through the queue. In the off chance that fails, the intent is that if Listener is disposed, it won't be used anymore, so GC can collect ALL the buffers. Best part of having an instance buffer pool instead of a static one.